product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -11,7 +11,7 @@ On Linux, the `toolkit.properties` file is located in `/usr/edb/migrationtoolkit
 
 On Windows, the file is located in `C:\Program Files\edb\mtk\etc`.
 
-On Mac OS, the file is located in `/Library/edb/mtk/etc`.
+On macOS, the file is located in `/Library/edb/mtk/etc`.
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.